Scalafmt 3.0.0-RC6 with Scala3 dialect has issues with implicit inline def

This template is a guideline, not a strict requirement.
- **Version**: 3.0.0-RC6
- **Integration**: sbt-scalafmt
- **Configuration**:
```
runner.dialect = scala3
version=3.0.0-RC6
maxColumn=120
project.git = true
```

## Steps

Given code like this:
```scala
implicit inline def qm: QueryMeta[PersonName, String] = { //hellooo
  queryMeta[PersonName, String](
    quote {
      (q: Query[PersonName]) => q.map(p => p.name)
    }
  )((name: String) => PersonName(name))
}
```

When I run scalafmt like this:

```bash
sbt test:scalafmt
```

## Problem

Scalafmt format fails with the following error
```
[error] /home/mdedetrich/github/protoquill/quill-sql/src/test/scala/io/getquill/QueryMetaTest.scala:66: error: => expected but def found
[error]       implicit inline def qm: QueryMeta[PersonName, String] = { //hellooo
[error]                       ^: /home/mdedetrich/github/protoquill/quill-sql/src/test/scala/io/getquill/QueryMetaTest.scala
```

## Expectation

scalafmt to format code correctly without any errors

## Workaround

None apart from deleting the code

## Notes

Here is the original code https://github.com/getquill/protoquill/blob/d095b54d4e267ed0f2a1e6a60b6f0aa16b122d99/quill-sql/src/test/scala/io/getquill/QueryMetaTest.scala#L66-L72

This seems to be just a case where the scala3 dialect isn't aware of a new case that is present in Scala3, it may also be an issue with scalameta not being up to date enough to handle this case?

--- a/project/Dependencies.scala
+++ b/project/Dependencies.scala
@@ -6,7 +6,7 @@ import org.portablescala.sbtplatformdeps.PlatformDepsPlugin.autoImport._
 
 object Dependencies {
   val metaconfigV = "0.9.14"
-  val scalametaV  = "4.4.24"
+  val scalametaV  = "4.4.26"
   val scalacheckV = "1.15.4"
   val coursier    = "1.0.3"
   val munitV      = "0.7.27"

--- a/scalafmt-tests/src/test/resources/scala3/InlineTransparent.stat
+++ b/scalafmt-tests/src/test/resources/scala3/InlineTransparent.stat
@@ -125,7 +125,12 @@ inline def repro[T <: Int with Singleton, U <: Int with Singleton]: Int =
   }
 }
 >>>
-test does not parse
 "static meta" - {
-      implicit inline def qm: QueryMeta[PersonName, String] = { //hellooo
-                    ^
+  implicit inline def qm: QueryMeta[PersonName, String] = { //hellooo
+    queryMeta[PersonName, String](
+      quote { (q: Query[PersonName]) =>
+        q.map(p => p.name)
+      }
+    )((name: String) => PersonName(name))
+  }
+}

--- a/scalafmt-tests/src/test/resources/scala3/InlineTransparent.stat
+++ b/scalafmt-tests/src/test/resources/scala3/InlineTransparent.stat
@@ -114,3 +114,18 @@ inline def repro[T <: Int with Singleton, U <: Int with Singleton]: Int =
         case _: true  => 100
         case _: false => 101
     case _ => 0
+<<< #2640
+"static meta" - {
+  implicit inline def qm: QueryMeta[PersonName, String] = { //hellooo
+    queryMeta[PersonName, String](
+      quote {
+        (q: Query[PersonName]) => q.map(p => p.name)
+      }
+    )((name: String) => PersonName(name))
+  }
+}
+>>>
+test does not parse
+"static meta" - {
+      implicit inline def qm: QueryMeta[PersonName, String] = { //hellooo
+                    ^

